Upgrade icons

basham opened the pull request.

levimcg commented:
@basham This look excellent! Love all the new icons 😄. Re: breaking changes. I'm fine with this being a major release and moving to `2.0.0`. We can make that clear when we send out our communications to the larger community.

I think a question we might need to figure out when we meet next is how to handle the current docs (Eleventy site). We've got the new icons docs section on the Rivet site in test are just about ready to launch it. The last step is updating to the new (2.0.0) release and then launching the site. I can share those docs when we get together in the next couple of weeks so we can sort out updating/adding links to the new docs on the Rivet site here in this repo.

basham commented:
I'm fine with breaking changes as well. I'd like to not maintain two documentation sites, but we still need a way to test the icons after any changes. If that can't easily be done with the new documentation site, then perhaps this Eleventy site should remain. However, we could remove the `gh-pages` branch and not make it a public site. It could instead be used for development and testing the icon set itself.

basham commented:
@levimcg I've made name changes, after consulting with Jenn. I've upgraded deps. It is ready for review.

## All breaking changes in this PR

Renamed icons:
- `audio-on` to `audio`
- `exclamation` to `exclamation-mark`
- `transfer-alternate` to `transfer-alt`
- `users` to `user-group`
- `users-solid` to `user-group-solid`

Removed icons:
- `users-group-solid`

levimcg reviewed (approved):
Everything looks good to me. Thanks for all the work on this! 🙏🏻  I'll get the release PR open here in a bit.
